snuba/migrations/snuba_migrations/profiles/0001_profiles.py

```diff
@@ -0,0 +1,88 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import UUID, Column, DateTime, NamedTuple, String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+columns: List[Column[Modifiers]] = [
+    # primary key
+    Column("organization_id", UInt(64)),
+    Column("project_id", UInt(64)),
+    Column("transaction_id", UUID()),
+    Column("received", DateTime()),
+    # profiling data
+    Column("profile", String(Modifiers(codecs=["LZ4HC(9)"]))),
+    Column("symbols", String(Modifiers(codecs=["LZ4HC(9)"]))),
```

Comment:
Is it safe to assume that these are non-searchable fields and would only be used for the use case where a user wants to do a lookup by a specific profile?

Reply:
They are non-searchable fields. They would be used when a user wants to lookup a specific profile but also when we would do an aggregation (for example, trying to profile a specific transaction on all iPhone 11 in the last week). We'd load those fields into our aggregation service and aggregate.

In the future, we hope to structure that into its own table and run aggregations directly in ClickHouse.